src/components/Api.js

```diff
@@ -0,0 +1,179 @@
+export class Api {
+  constructor(config) {
+    this.url = config.baseUrl;
+    this.headers = config.headers;
+  }
+
+  getInitialCards() {
+    return fetch(`${this.url}/cards`, {
+      headers: {
+        authorization: "fe7e07a4-81c5-490b-807b-e6a7cec619a0",
+      },
+    }).then((res) => {
```

Comment:
Needs correction: The code for processing server response gets duplicated. You should declare a method and reuse it

```js
_processResponse = (res) => { 
      if (res.ok) { 
        return res.json(); 
      } 
      return Promise.reject(`Error: ${res.status}`); 
}

getInitialCards() { 
  ...
  .then(this._processResponse)
}
```

Reply:
I updated the code as recommended. Thank you.